fondant/compiler.py

```diff
@@ -106,7 +117,7 @@ def _generate_spec(self, pipeline: Pipeline) -> dict:
             command = ["--metadata", json.dumps(asdict(metadata))]
 
             # add in and out manifest paths to command
-            command.extend(["--output_manifest_path", f"{path}/manifest.txt"])
+            command.extend(["--output_manifest_path", f"{path}/manifest.json"])
```

Comment:
This still overwrites the manifest for every component. We should still address this, but can be taken up in a separate PR.

Reply:
will fix in another PR